### PR TITLE
Add codFonte.

### PR DESCRIPTION
**Pull request by RicardoBPedro**
Colaborando...

### CODE REVIEW

**Comment by GoranMinov** [comment]
@casey-collab

**Comment by pmallory** [comment]
@casey-collab can you take a look?

File: portugues/colabora/codFonte.md [hunk 1]
```diff
@@ -0,0 +1 @@
+E então...
```
Comment (casey-collab) [inline portugues/colabora/codFonte.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**